git-gui/po/README

```diff
@@ -4,7 +4,7 @@ Localizing git-gui for your language
 This short note is to help you, who reads and writes English and your
 own language, help us getting git-gui localized for more languages.  It
 does not try to be a comprehensive manual of GNU gettext, which is the
-i18n framework we use, but tries to help you get started by covering the
+i18n framework we use but tries to help you get started by covering them
```

Comment:
I am no expert about using commas in English, but "them basics" sounds very colloquial to me.